src/styles/main.scss

```diff
@@ -20,9 +20,11 @@ body {
   text-align: center;
   vertical-align: center;
   user-select: none;
+  transition: all 0.5s ease-in-out;
```

Comment:
Consider specifying which properties should be transitioned instead of using 'all' to improve performance.